crates/mithril-oxide-sys/cpp/IR/BuiltinTypes.cpp

```diff
@@ -0,0 +1,24 @@
+#include "BuiltinTypes.hpp"
+
+#include <memory>
+#include <mlir/IR/BuiltinAttributes.h>
+
+namespace mithril_oxide_sys
+{
```

Comment:
Since a `namespace` declaration is always preceded and followed by a blank line, there's no readability gain in placing `{` in the new line. This is one of the few exceptions to placing `{` on the next line.